estimatey-api/Estimatey.Application/WorkItems/Queries/ListFeaturesQuery.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Estimatey.Application.Common.AppRequests;
+using Estimatey.Application.Common.Interfaces;
+using Estimatey.Application.WorkItems.Dtos;
+using Microsoft.EntityFrameworkCore;
+
+namespace Estimatey.Application.WorkItems.Queries;
+
+public record ListFeaturesQuery
+{
+    public int ProjectId { get; init; }
+}
+
+public class ListFeaturesQueryHandler : IRequestHandler<ListFeaturesQuery, List<FeatureSummary>>
+{
+    private readonly IApplicationDbContext _dbContext;
+
+    public ListFeaturesQueryHandler(IApplicationDbContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+
+    public async Task<AppResponse<List<FeatureSummary>>> Handle(
+        ListFeaturesQuery query,
+        CancellationToken cancellationToken)
+    {
+        var featureSummaries = await _dbContext.Features
+            .Where(_ => _.ProjectId == query.ProjectId)
+            .Include(_ => _.UserStories)
+                .ThenInclude(_ => _.Tickets)
+            .Select(entity => FeatureSummary.MapFromEntity(entity))
+            .ToListAsync(cancellationToken);
+
+        return new(200, featureSummaries.OrderBy(_ => _.SortOrder).ToList());
```

Comment:
I think you should be able to do the `OrderBy` in the EF query as long as it's after the `Select`